pom.xml

```diff
@@ -1070,12 +1068,12 @@
             <dependency>
                 <groupId>org.slf4j</groupId>
                 <artifactId>slf4j-api</artifactId>
-                <version>1.7.26</version>
+                <version>1.7.28</version>
             </dependency>
             <dependency>
                 <groupId>org.slf4j</groupId>
                 <artifactId>slf4j-simple</artifactId>
-                <version>1.7.26</version>
+                <version>1.7.28</version>
```

Comment:
What about extracting this into a `<property/>`?

Reply:
Good idea. I add a new commit.